src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,275 @@
+2. Grants them a Influence conversion aura around themselves.
+3. Grants them a conversion ability that allows them to covertly flip people to Rev.
+
+### Removal of MindShield implants being orderable from Cargo
+
+MindShields are metagameable and make it extremely unfun to play as a Head Rev. They have no interesting interactions; they’re just a hand-wave for why some people are completely immune to a game mechanic, and whilst that’s utterly required for round flow (allowing Sec and the Heads to trust each other) it is a magic solve-everything button which leads to a total wall in design.
+
+At best, a small number of MindShields could be kept on-station to allow staff promotions, and if these run out then more can be begged for via asking CentComm nicely.
+
+### Removal of Flashes being able to convert people
+
+Flashes are metagameable, confusing for newbies, hard to balance, and enforce too much reliance on manufacture by Science by the Revs for effective round flow. They also have no narrative justification beyond it being an Ur-mode from the dawn of SS13. There are people playing SS14 that are younger than the only good _Men In Black_ movie.
+
+### Addition of Influence as a tracked player stat, and associated gameplay features
+
+Influence is a value that each player carries. MindShielded players are always value 0. Influence is raised by a few routes (exposure to Head Revs, propaganda and one or two restricted drugs like Mindbreaker Toxin).
+
+### Influence ticks up when exposed to sources and slowly ticks down when not. There is a max Influence level.
+
+Addition of NT and Revolutionary posters and propaganda, that both sides, and traitors, can put up and take down across the station.
+
+These posters can be printed across the station by both sides and gives Security Cadets, Lawyers and bored Passengers something to do, even on non-Rev rounds, and a reason for Sec to patrol (imagine!). They’re an DOT effect with a max cap per item, and add or subtract Influence from those who see them.
+
+### Addition of Loyalty Loudspeakers in Sec
+
+These Loyalty Loudspeakers are audio emitters that reduce a player’s Influence level, even if they’re converted. If a player who is converted de-converts, their Influence immediately drops to zero. This is a new machine that requires quite a bit of power to run. A button is added to the Warden’s office to disable and enable all loudspeakers. The loudspeakers play calming, smooth jazz.
+
+### Head Revs can use an implant ability to silently convert characters with high enough Influence to being a Rev.
+
+This replaces flashing outright, and makes conversion a stealthy game. Revs do not know who the Head Revs are unless the Head Rev outs themselves.
+
+### Being a Rev has a tell, similar to some old SS13 conversion modes.
+
+Being a Rev means you look slightly dazed and confused, which can be hidden by hiding your eyes. This effect will also be applied to some drugs in game like Mindbreaker Toxin.
```

Comment:
This is extremely gameable, as it strongly encourages shift-clicking everyone in sight for the purpose of validhunting, and discourages revs from trying to blend in

Reply:
"which can be hidden by hiding your eyes."

Reply:
There needs to be SOME way to confirm a headrev.

Reply:
See the Role Outline for Revolutionary:

```Converted Revolutionaries have an obvious trait that can be inspected:

“They appear slightly dazed and confused”.

This state is shared with imbiding some contraband chemicals, like Mindbreaker Toxin: being dazed and confused itself is not evidence of the round type being Revolutionaries.```

You are not valid if you're seen "dazed and confused" but it should be suspicious, and you can prevent it by wearing eye protection.

Reply:
Please read the last sentence of the above post as "please, God, somebody think of a better idea than eyes, because I'm 99% sure I've stolen that idea from a Cultist implementation on TG or Goon and it wasn't great there"